
github.com/BYVoid/OpenCC/pull/934
github.com/BYVoid/OpenCC/commit/72cae18


From 72cae18cfe4272f2b11c9ec1c44d6af7907abcab Mon Sep 17 00:00:00 2001
From: Peng Wu <alexepico@gmail.com>
Date: Thu, 13 Feb 2025 00:15:10 +0800
Subject: [PATCH] Fix build for gcc 15 (#934)

Co-authored-by: Peng Wu <pwu@redhat.com>
---
 src/SerializedValues.hpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/SerializedValues.hpp b/src/SerializedValues.hpp
index c17210ba7..f7e5bb264 100644
--- a/src/SerializedValues.hpp
+++ b/src/SerializedValues.hpp
@@ -18,6 +18,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include "Common.hpp"
 #include "SerializableDict.hpp"
 

